PiRemote/PiRemote/MainUser.swift

```diff
@@ -5,25 +5,60 @@
 //  Created by Victor Anyirah on 2/22/17.
 //  Copyright © 2017 JLL Consulting. All rights reserved.
 //
-
+import UIKit
 import Foundation
 
-class MainUser {
+class MainUser{
     
     static let sharedInstance = MainUser()
    
     private init(){}
-
+    
     // Remot3.it
     var email : String?
-    var apiKey: String?
-    var token: String?
+    
+    // Weird. Apparently this is the global API key and they specified they planned on having individual keys in the future..but I didn't find anything
+    // in the docs
+    // see : http://docs.weaved.com/v21/discuss/577bde161cf3cb0e0048e357
+    var apiKey: String = "WeavedDemoKey$2015"
```

Comment:
I couldn't find anything either. Have a hunch that it's gonna update anytime soon 😷